modules/virt-measuring-latency-vm-secondary-network.adoc

```diff
@@ -0,0 +1,229 @@
++
+.Example input config map
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubevirt-latency-checkup-example-config
+  namespace: k8s-checkup-framework
+data:
+  spec.image: registry:5000/kubevirt-latency-checkup:latest
+  spec.timeout: 5m
+  spec.clusterRoles: |
+    nodes-reader
+    network-reader
+    kubevirts-reader
+    kubevirt-vmis-manager
+  spec.param.nad_namespace: "nad-namespace"
+  spec.param.nad_name: "br1"
+  spec.param.nad_namespace: "nad-namespace" <1>
+  spec.param.nad_name: "br1"  <2>
+  spec.param.source_node: node01  <3>
+  spec.param.targer_node: node02  <4>
+  spec.param.sample_duration_seconds: 300  <5>
+  spec.param.max_desired_latency_milliseconds: 50  <6>
+----
+<1> The namespace where the `NetworkAttachmentDefinition` object resides.
+<2> The name of the `NetworkAttachmentDefinition` object.
+<3> Optional: Check latency from this node to the target node. When specified, the VM starts with an affinity rule configured.
+<4> Optional: Check latency from the source node to this node. When specified, the VM starts with an affinity rule configured.
+<5> Optional: The duration of the latency check, in seconds.
+<6> Optional: The maximum desired latency, in milliseconds, between the virtual machines. If the measured latency exceeds this value, the check fails.
+
+. Create the config map in the framework’s namespace:
++
+[source,terminal]
+----
+$ oc apply -f <latency-config-map>.yaml
+----
+
+. Create a `Job` object to run the checkup:
++
+.Example job manifest
+[source,yaml]
+----
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: checkup-framework
+  namespace: k8s-checkup-framework
+spec:
+  backoffLimit: 0
+  template:
+    spec:
+      serviceAccountName: checkup-framework-sa
+      restartPolicy: Never
+      containers:
+      - name: checkup-framework
+        resources:
+        image: registry:5000/checkup-framework:latest
+        env:
+        - name: CONFIGMAP_NAMESPACE
+          value: k8s-checkup-framework
+        - name: CONFIGMAP_NAME
+          value: kubevirt-latency-checkup-example-config
+----
+
+. Apply the `Job` manifest. The checkup uses the ping utility to verify connectivity and measure latency.
++
+[source,terminal]
+----
+$ oc apply -f <latency-job>.yaml
+----
+
+. Wait for the job to complete:
++
+[source,terminal]
+----
+$ oc wait --for=condition=complete job.batch/checkup-framework -n k8s-checkup-framework
+----
+
+. Review the results of the latency checkup by retrieving the status of the `ConfigMap` object. If the measured latency is greater than the value of the `spec.param.max_desired_latency_milliseconds` attribute, the checkup fails and returns an error.
++
+.Example output config map (success)
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubevirt-latency-checkup-example-config
+  namespace: k8s-checkup-framework
+...
+  status.succeeded: “true”
```

Comment:
The curly quotation marks might create problems in the output. I recommend you try to replace them with `"`. I always assumed that code blocks did not support this kind of formatting -- until I saw someone's PR where a code block turned `--` into an en-dash.